blockchain/blocks/src/block.rs

```diff
@@ -3,9 +3,8 @@
 
 use super::BlockHeader;
 use cid::Cid;
-use encoding::{de::Deserializer, ser::Serializer};
+use encoding::tuple::*;
```

Comment:
Can you import the specific stuff instead of everything? Just to keep consistancy

Reply:
Yeah I don't mind either way, the reason I did it like this is that `serde_tuple` for whatever reason requires `serde_tuple` itself to be in scope as well

Reply:
can do `::{self, ..}` in that case, I don't mind the wildcard myself, just mentioning

Reply:
Oh, I see. Yeah, Wildcarding should be ok in this case then.